src/ListL.php

```diff
@@ -4,8 +4,6 @@
 
 namespace Marcosh\LamPHPda;
 
-use ArrayIterator;
-use Iterator;
 use IteratorAggregate;
```

Comment:
could we remove also `IteratorAggregate`, then?

Reply:
Sadly no. When I remove it, Psalm is failing.